org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/adwindow/ADSortTab.java

```diff
@@ -79,7 +80,11 @@
 	/**
 	 * 
 	 */
-	private static final long serialVersionUID = -2238411612673317537L;
+	private static final long serialVersionUID = 4302282658814599752L;
+
+	public ADSortTab()

```

Comment:
what's the use of this new constructor ?

Reply:
It used when we get object for osgi service in Extensions.getADTabPanel. We later call init method after retrieving object of implementer class.